alibi_detect/utils/pydantic_v1/fields/tests/test_import_pydantic_v1_fields.py

```diff
@@ -0,0 +1,14 @@
+def test_import():
```

Comment:
I'm not sure we really need to test this? Isn't it pydantic functionality? All we really do is import differently

Reply:
Code coverage started to fail as this introduced new code + some branching, in that new code, so I added a relatively small unit test to ensure that code coverage doesn't drop below the accepted level